### DISCUSSION TITLE
How do i calculate security hash on the frontend?

### DISCUSSION THREAD

**Discussion by ZeroCoolHacker** [danirus/django-comments-xtd#240] [Q&A] [answered]
I am trying to use it as a rest api. But it always throws security hash related errors. How should i be calculating what to pass there?

**Comment by danirus** [comment, marked as answer]
The post api entry at `<comments-mount-point>/api/comment/` expects the same fields as when you post using the regular form. The regular form expects the security fields among which there is a `security_hash` field. However you can use the alternative solution of authorizing the request. I explained it [here](https://django-comments-xtd.readthedocs.io/en/latest/webapi.html#post-a-new-comment). Please read it and if you have problems let me know.
